cinch/roles/jenkins_slave/tasks/main.yml

```diff
@@ -69,6 +69,14 @@
     mode: 0644
   notify: restart swarm
 
+- name: inject additional variables to swarm's environment
+  lineinfile:
+    dest: /etc/sysconfig/jenkins_swarm
+    line: "{{ item }}"
+    state: present
+  with_items: "{{ jswarm_environment }}"
+  notify: restart swarm
```

Comment:
Rather than upload a file, then modify it, can you move this into the jenkins_swarm template file and have it iterate over the list of variables directly in the template?